File: Assignment 7/Problem5.java
//Write a program which accept number from user and return difference between summation of even digits and summation of odd digits.

/*
  Input : 2395
  Output : -15
  
  Input : 1018
  Output : 6
*/

import java.util.Scanner;

public class Problem5 
{
    public static void main(String [] arg)
    {
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter Number : ");
        int iNo = sc.nextInt();
        int evenSum = 0;
        int oddSum = 0;
        int totalDiff = 0;

        while(iNo != 0)
        {
            int rem = iNo%10;
            if(rem % 2 == 0)
            {
                evenSum = evenSum + rem;
            }
            else 
            {
                oddSum = oddSum + rem;
            }
            iNo /= 10;
        }
        totalDiff = evenSum - oddSum;
        System.out.println(totalDiff);
    }   
}

File: Assignment 2/problem5.c
// Accept number from user and check whether number is even or odd.
#include <stdio.h>

#define TRUE 1
#define FALSE 0

typedef int BOOL;

BOOL ChkEven(int iNo)
{
    if (iNo % 2 == 0)
    {
        printf("Number is even");
    }
    else
    {
        printf("Number is odd");
    }
}

int main()
{
    int iValue = 0;
    BOOL bRet = FALSE;

    printf("Enter number");
    scanf("%d", &iValue);

    bRet = ChkEven(iValue);

    return 0;
}